pkg/webhook/handler/validator/secret.go

```diff
@@ -0,0 +1,81 @@
+package validator
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	corev1 "k8s.io/api/core/v1"
+
+	"github.com/juicedata/juicefs-csi-driver/pkg/config"
+	"github.com/juicedata/juicefs-csi-driver/pkg/juicefs"
+)
+
+type SecretValidator struct {
+	jfs juicefs.Interface
+}
+
+var _ Validator[corev1.Secret] = &SecretValidator{}
+
+func NewSecretValidator(jfs juicefs.Interface) *SecretValidator {
+	return &SecretValidator{
+		jfs: jfs,
+	}
+}
+
+func (s *SecretValidator) Validate(ctx context.Context, secret corev1.Secret) error {
+	secretsMap := make(map[string]string)
+	if configs, ok := secret.Data["configs"]; ok {
+		err := s.ValidateConfigs(string(configs[:]))
+		if err != nil {
+			return err
+		}
+	}
+	if envs, ok := secret.Data["envs"]; ok {
+		err := s.ValidateEnvs(string(envs[:]))
+		if err != nil {
+			return err
+		}
+	}
+
+	for k, v := range secret.Data {
+		secretsMap[k] = string(v[:])
+	}
+	jfsSetting, err := s.jfs.Settings(ctx, "", secretsMap, nil, nil)
+	if err != nil {
+		return err
+	}
+
+	// ignore ce secrets
+	if !jfsSetting.IsCe {
+		_, err := s.jfs.AuthFs(ctx, secretsMap, jfsSetting, true)
```

Comment:
is there any way that we can "check" JuiceFS CE secrets? if there's any meaningful checks that we can do during validation webhook, this function can help CE users as well @zhijian-pro @SandyXSD